### PR TITLE
Updated reference

### PR DESCRIPTION
**Pull request by dannago**

### CODE REVIEW

File: sp800-63-3/sec8_references.md [hunk 1]
```diff
@@ -5,52 +5,62 @@
 
 *This section is informative.*
 
-<a name="EO13681"></a>[EO 13681] Executive Order 13681, *Improving the Security of Consumer Financial Transactions*, October 17, 2014, available at: <https://obamawhitehouse.archives.gov/the-press-office/2014/10/17/executive-order-improving-security-consumer-financial-transactions>.
+### 1. General references
```
Comment (jimfenton) [inline sp800-63-3/sec8_references.md]: Should be section 8.1 (no trailing dot)
references --> References

File: sp800-63-3/sec8_references.md [hunk 2]
```diff
@@ -5,52 +5,62 @@
...
-<a name="GPG44"></a>[GPG 44] UK Cabinet Office, Good Practice Guide 44, *Authentication and Credentials for use with HMG Online Services*, August 8, 2016, available at: <https://www.ncsc.gov.uk/guidance/authentication-and-credentials-use-hmg-online-services-gpg-44>.
 
-<a name="GPG45"></a>[GPG 45] UK Cabinet Office, Good Practice Guide 45, *Identity proofing and verification of an individual*, November 3, 2014, available at: <https://www.gov.uk/government/publications/identity-proofing-and-verification-of-an-individual>.
+<a name="steiner"></a>[Steiner] Steiner, Peter. "On the Internet, nobody knows you're a dog", *The New Yorker*, July 5, 1993.
 
 <a name="STORK2.0"></a>[STORK 2.0] European Union, *Secure idenTity acrOss boRders linKed 2.0*, 2014, available at: <https://www.eid-stork2.eu/>.
 
-<a name="eIDAS"></a>[eIDAS] European Union, *REGULATION (EU) No 910/2014 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL*, July 23,  2014, available at: <http://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2014.257.01.0073.01.ENG>.
+### 2. Standards
```
Comment (jimfenton) [inline sp800-63-3/sec8_references.md]: 8.2

File: sp800-63-3/sec8_references.md [hunk 3]
```diff
@@ -5,52 +5,62 @@
...
+<a name="ISO29003"></a>[ISO 29003] International Standards Organization, ISO/IEC DTS 29003 *Information technology — Security techniques — Identity proofing*.
 
-<a name="steiner"></a>[Steiner] Steiner, Peter. "On the Internet, nobody knows you're a dog", *The New Yorker*, July 5, 1993.
+<a name="ISO29115"></a>[ISO 29115] International Standards Organization, ISO/IEC 29115 *Information technology — Security techniques — Entity authentication assurance framework*, April 1, 2013, available at: <http://www.iso.org/iso/home/store/catalogue_tc/catalogue_detail.htm?csnumber=45138>.
 
-<a name="NIST-RMF"></a>[NIST RMF] Risk Management Framework Overview, available at <http://csrc.nist.gov/groups/SMA/fisma/framework.html>.
+<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014,  Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
+
+### 3. NIST Special publications
```
Comment (jimfenton) [inline sp800-63-3/sec8_references.md]: 8.3
publications --> Publications

File: sp800-63-3/sec8_references.md [hunk 4]
```diff
@@ -5,52 +5,62 @@
...
+
+<a name="SP800-37"></a>[SP 800-37] NIST Special Publication 800-37 Revision 1, *Guide for Applying the Risk Management Framework to Federal Information Systems, A Security Life Cycle Approach*, February 2010 (updated June 5, 2014), <https://doi.org/10.6028/NIST.SP.800-37r1>.
+
+<a name="SP800-53A"></a>[SP 800-53A] NIST Special Publication 800-53A Revision 4, *Assessing Security and Privacy Controls in Federal Information Systems and Organizations, Building Effective Assessment Plans*, December 2014 (updated December 18, 2014), <https://doi.org/10.6028/NIST.SP.800-53Ar4>.
 
-<a name="bcp195"></a>[BCP 195] Sheffer, Y., Holz, R., and P. Saint-Andre, *Recommendations for Secure Use of Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS)*, BCP 195, RFC 7525, May 2015, <http://www.rfc-editor.org/info/bcp195>.
 
-<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014. Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
+### 4. Federal Information Processing Standards
```
Comment (jimfenton) [inline sp800-63-3/sec8_references.md]: 8.4

File: sp800-63-3/sec8_references.md [hunk 5]
```diff
@@ -5,52 +5,62 @@
...
+<a name="FIPS201"></a>[FIPS 201] Federal Information Processing Standard Publication 201-2, *Personal Identity Verification (PIV) of Federal Employees and Contractors*, August 2013,  <http://dx.doi.org/10.6028/NIST.FIPS.201-2>.
```
Comment (jimfenton) [inline sp800-63-3/sec8_references.md]: Double space after 2013,

File: sp800-63-3/sec8_references.md [hunk 6]
```diff
@@ -5,52 +5,62 @@
...
 
-<a name="Canada"></a>[Canada] Government of Canada, *Standard on Identity and Credential Assurance*, February 1, 2013, available at: <https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=26776>.
+<a name="ISO29003"></a>[ISO 29003] International Standards Organization, ISO/IEC DTS 29003 *Information technology — Security techniques — Identity proofing*.
 
-<a name="steiner"></a>[Steiner] Steiner, Peter. "On the Internet, nobody knows you're a dog", *The New Yorker*, July 5, 1993.
+<a name="ISO29115"></a>[ISO 29115] International Standards Organization, ISO/IEC 29115 *Information technology — Security techniques — Entity authentication assurance framework*, April 1, 2013, available at: <http://www.iso.org/iso/home/store/catalogue_tc/catalogue_detail.htm?csnumber=45138>.
 
-<a name="NIST-RMF"></a>[NIST RMF] Risk Management Framework Overview, available at <http://csrc.nist.gov/groups/SMA/fisma/framework.html>.
+<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014,  Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
```
Comment (jimfenton) [inline sp800-63-3/sec8_references.md]: double space before Available, and Available --> available.
Are we keeping the "available at" thing? looks like it was removed in another volume.

**Review by jimfenton — CHANGES_REQUESTED** [review]
Similar comments on references as other volumes

File: sp800-63-3/sec8_references.md (hunk 6 above)
Comment (RGalluzzo) [reply sp800-63-3/sec8_references.md]: Per Jim Foti, where it is a DOI link we should not have "available at." He seemed not to have a problem with using it where we did not have a DOI link. Not too heavy a lift I don't think to remove all of them. Up to you...

File: sp800-63-3/sec8_references.md [hunk 7]
```diff
@@ -50,9 +50,9 @@ available at: <https://cio.gov/wp-content/uploads/downloads/2014/03/Use_of_ESign
 
 <a name="ISO29115"></a>[ISO 29115] International Standards Organization, ISO/IEC 29115 *Information technology — Security techniques — Entity authentication assurance framework*, April 1, 2013, available at: <http://www.iso.org/iso/home/store/catalogue_tc/catalogue_detail.htm?csnumber=45138>.
 
-<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014,  Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
+<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014, available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
 
-### 3. NIST Special publications
+### 8.3 NIST Special publications
```
Comment (jimfenton) [inline sp800-63-3/sec8_references.md]: Still need publications --> Publications

**Review by jimfenton — APPROVED** [review]

**Pull request merged**